Parsers/ASimAuditEvent/Parsers/ASimAuditEventVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,183 @@
+            | extend
+                EventType = "Set",
+                Operation = strcat(temp_operation, " to ", NewValue, " for ID: ", temp_policyid),
+                Object = strcat("Policy ", temp_policyid),
+                ObjectType = "Policy Rule",
+                AdditionalFields = bag_pack("devices", temp_deviceid),
+                TargetDvcId = temp_deviceid;
+        let ParamsUpdated = allData
+            | where description_s startswith "Parameters updated"
+            | parse description_s with "Parameters updated for " temp_config1: string " (ID: " temp_configid1: string ") for policy " temp_policyname1: string " (ID: " temp_policyid1: string ")" temp_restmessage1: string
+            | parse description_s with "Parameters updated for " temp_config2: string " (ID: " temp_configid2: string ") for policy with ID " temp_policyid2: string
+            | extend
+                temp_operation = coalesce(temp_config1, temp_config2),
+                temp_configid = coalesce(temp_configid1, temp_configid2)
+            | extend
+                EventType = "Set", 
+                Operation = strcat("Parameters updated for ", temp_operation, " for policy ", temp_policyname1, tostring(split(temp_policyid2, "{")[0])),
+                Object = strcat("Policy ", coalesce(temp_policyname1, temp_policyid2)),
+                ObjectType = "Policy Rule",
+                AdditionalFields = bag_pack("config id", temp_configid);
+        let Reputation = allData
+            | where description_s has_cs "Reputation"
+            | parse description_s with "User " * " " temp_type1: string " Reputation" * " for Organization ID " temp_orgid1: string " of type " temp_reptype1: string " to " temp_list1: string " with content: " temp_content1: string " | " temp_restmessage1: string
+            | parse description_s with "User " * " " temp_type2: string " Reputation" * " for Organization ID " temp_orgid2: string ": " temp_content2: string " | " temp_restmessage2: string
+            | extend
+                temp_type = coalesce(temp_type1, temp_type2),
+                Object = iff(isnotempty(temp_reptype1), strcat("Reputation Override of type ", temp_reptype1), "Reputation Override"),
+                ObjectType = "Configuration Atom"
+            | lookup EventTypeLookup on temp_type
+            | extend Operation = strcat(temp_type, " ", Object)
+            | extend AdditionalFields = bag_pack("reputation value", coalesce(temp_content1, temp_content2));
+        union
+            Enabled,
+            Set,
+            AlertNotify,
+            CustomRole,
+            Policy,
+            Changed,
+            ParamsUpdated,
+            Reputation
+        | extend
+            EventStartTime = unixtime_milliseconds_todatetime(eventTime_d),
+            EventSeverity = coalesce(EventSeverity1, "Informational"),
+            AdditionalFields = bag_merge(AdditionalFields, bag_pack("flagged", flagged_b))
```

Comment:
for authentication we used this field to set Severity why not here?

Reply:
As for AuditActivities, it's always false. In case of Authentication, flagged_b=true only when login is failed due to incorrect password.